modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a Subscription
+
+You can create a Subscription to connect a Service or other event sink to a Channel.
+The following procedure shows how to create a Subscription object that uses the default InMemoryChannel type Channel.
+
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+// * If you want to use other Channel types with Subscriptions...
+// TODO: Create docs for these, link Kafka etc?
+
+.Procedure
+
+* Create a Subscription object that uses the default InMemoryChannel type Channel by entering the following:
++
+[source,yml]
+----
+$ oc  create -f - <<EOF
+apiVersion: messaging.knative.dev/v1
+kind: Subscription
+metadata:
+  name: my-subscription
+spec:
+  channel:
+    apiVersion: messaging.knative.dev/v1
+    kind: InMemoryChannel
+    name: default
+  subscriber:
+    uri: <SERVICE-URI> <1>
```

Comment:
I believe the "uri" should only be used if the subscriber isn't something a kubernetes resource by itself, as it is safer to use "ref", eg.

```
  subscriber:
    ref:
      apiVersion: serving.knative.dev/v1
      kind: Service
      name: foobar
```